optuna/samplers/_cmaes.py

```diff
@@ -484,6 +519,26 @@ def _init_optimizer(
                 population_size=population_size,
             )
 
+        if self._with_margin:
+            steps = []
+            for dist in trans._search_space.values():
+                assert isinstance(dist, (IntDistribution, FloatDistribution))
+                # Set step 0.0 for continuous search space.
+                steps.append(0.0 if dist.step is None else dist.step)
```

Comment:
nit:

```suggestion
            # Set step 0.0 for continuous search space.
            steps = [dist.step or 0.0 for dist in trans._search_space.values()]
```

Reply:
This does not work because `BaseDistribution` does not have the "step" attribute (mypy throws an error).

Reply:
Ah, I see.